lesson16/main.js

```diff
@@ -59,23 +59,79 @@ const renderTabs = (newsGenres) => {
   tabs[0].classList.add("active");
 };
 
-/**
- * this function will be modified to render articles, comments, an image and icons.
- * @param newsGenres
- */
-const renderArticles = (newsGenres) => {
+const createNewIcon = ({ date }) => {
+  const articleDate = new Date(date);
+  const today = new Date();
+  const newIcon = document.createElement("i");
+  const calculateToInteger = 1000 * 60 * 60 * 24;
+  newIcon.className = "fa-regular fa-bell";
+  if ((today - articleDate) / calculateToInteger < 4) {
+    newIcon.classList.add("new");
+    console.log(newIcon);
```

Comment:
Q
Do you need it here? I want to know the reason why it is necessary.

Reply:
I forgot to remove console.log and it was not necessary, it was a good call
Removed console.log() 
please see the commit 
[22dd075](https://github.com/Saki-Ota/JS_practice/pull/24/commits/22dd075120d9c4ff534953a30a36506749a8aac3)